DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>captial -&gt; capital. Also I think "in some Gaia table" should be "in some Gaia tables" or "in the Gaia table"? But I'm not sure I understood for sure if this correction is needed, I've never used the Gaia table(s) before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>